lib/compiler-cranelift/src/compiler.rs

```diff
@@ -565,6 +588,39 @@ fn mach_trap_to_trap(trap: &MachTrap) -> TrapInformation {
     }
 }
 
+#[derive(Clone)]
+struct ProgressContext {
```

Comment:
Can we use a common `CompilerProgressContext` type living in `wasmer-compiler` crate?